dev_tools/shell_tools.py

```diff
@@ -290,19 +290,19 @@ def output_of(*cmd: Optional[str], **kwargs) -> str:
 
     Args:
         *cmd: Components of the command to execute, e.g. ["echo", "dog"].
-        **kwargs: Extra arguments for asyncio.create_subprocess_shell, such as
+            Arguments set to None are disregarded in command execution.
+        **kwargs: Extra arguments for the shell_tools.run function, such as
             a cwd (current working directory) argument.
 
     Returns:
-        A (captured output, captured error output, return code) triplet. The
-        captured outputs will be None if the out or err parameters were not set
-        to an instance of TeeCapture.
+        str: The standard output of the command with the last newline removed.
```

Comment:
Are stderr and exit code no longer needed?

Reply:
The original docstring was incorrect, perhaps a result of copy-paste.
The output_of() function returns/returned only the stdout string.